contrib/Makefile.autosetup

```diff
@@ -10,7 +10,9 @@ clean-contrib:
 install-contrib:
 	$(INSTALL) -d -m 755 $(DESTDIR)$(docdir)/samples
 	for f in $(SAMPLES); do \
-		$(INSTALL) -m 644 $(SRCDIR)/contrib/$$f $(DESTDIR)$(docdir)/samples || exit 1; \
+		sed -e 's!/usr/libexec!$(libexecdir)!g' $(SRCDIR)/contrib/$$f > $(SRCDIR)/contrib/$$f.tmp; \
+		$(INSTALL) -m 644 $(SRCDIR)/contrib/$$f.tmp $(DESTDIR)$(docdir)/samples || exit 1; \
```

Comment:
I think you want `$(INSTALL) -m 644 $(SRCDIR)/contrib/$$f.tmp $(DESTDIR)$(docdir)/samples/$$f`

Reply:
Aha, good catch, fixed.